plugins/wacom-raw/wacom-raw.quirk

```diff
@@ -2,8 +2,8 @@
 # need to have an extra GUID of WacomAES or WacomEMR added so that the flash
 # constants are set correctly.
 
-# Test purpose
-[HIDRAW\VEN_056A&DEV_5F0F]
+#This device has USB I/F
```

Comment:
needs a space after the `#` please. Is it possible to name the laptop model yet? It's much easier for debugging.

Also, can you rebase against `main` and then force push this branch please, it'll make the history cleaner. Thanks.